services/core-web/src/components/mine/NoticeOfWork/MineNoticeOfWorkTable.js

```diff
@@ -111,12 +111,12 @@ export class MineNoticeOfWorkTable extends Component {
     {
       title: "NoW No.",
       dataIndex: "nowNum",
-      sortField: "tracking_number",
+      sortField: "now_number",
       render: (text, record) => (
-        <Link to={router.NOTICE_OF_WORK_INITIAL_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
+        <Link to={router.VIEW_NOTICE_OF_WORK_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
       ),
```

Comment:
same question here.

Reply:
It crossed my mind but I see no harm in leaving it either. I like having the links in general.